project/frontend/src/app/page.tsx

```
@@ -0,0 +1,230 @@
+           }}
+         >
+           Join Our Community
+         </h3>
+         <iframe
+           src="https://discord.com/widget?id=1326492630585966662&theme=dark"
+           width="350"
+           height="500"
+           allowtransparency="true"
+           frameBorder="0"
+           sandbox="allow-popups allow-popups-to-escape-sandbox allow-same-origin allow-scripts"
+         ></iframe>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)